lib/slim_fast/parser.ex

```diff
@@ -101,7 +101,7 @@ defmodule SlimFast.Parser do
              |> String.split(~r/^(-|==|=)/)
              |> List.last
              |> String.lstrip
-    inline = inline or String.starts_with?(input, "=")
+    inline = inline or String.starts_with?(input, "=") or String.starts_with?(script, "if")
```

Comment:
Related to my initial comment, I just had a look at the EEx docs and that confirmed (as you already knew) that it's different enough from Ruby ERB that it's probably tricky to get the same behaviour:

> All expressions that output something to the template must use the equals sign (=). Since everything in Elixir is a macro, there are no exceptions for this rule. For example, while some template languages would special- case if clauses, they are treated the same in EEx and also require = in order to have their result printed

http://elixir-lang.org/docs/v1.0/eex/EEx.html

Since EEx has made this design choice (rather than attempt to be more similar to e.g. PHP or ERB), it makes sense to me for us to weigh "true to Elixir" higher than "same as Ruby Slim".

This will make the transition a bit more difficult if you're a Ruby dev switching to Elixir, but I think good documentation and maybe good error messages/warnings could help that.

So in this particular case, with `- if`, maybe instead of allowing it, we could show a warning or something, and also make sure the doc examples have an `= if`, and maybe a section like "Differences to Ruby Slim" where we could explain the differences and the reason why.